### ISSUE TITLE
Internal usage of `figure` leads to accessibility errors

### ISSUE THREAD

**Issue by Marcono1234** [typst-community/glossarium#196]
### Version
- Typst 0.15.0
- Glossarium 0.5.10

### Description
When compiling a Typst project which uses Glossarium using `--pdf-standard a-2a` or `--pdf-standard ua-1`, Typst reports an error in the internal Glossarium code:
```
error: PDF/UA-1 error: missing alt text
     ┌─ @preview/glossarium:0.5.10/themes/default.typ:1343:13
     │  
1343 │       return [#figure(
     │ ╭──────────────^
1344 │ │         kind: __glossarium_entry,
1345 │ │         supplement: "",
1346 │ │         numbering: none,
1347 │ │       )[]#label(label-key)]
     │ ╰─────────^
     │  
     = hint: make sure your images and equations have alt text
```

Since this `figure` is only used for technical reasons apparently (?) maybe it would be best to suppress this error, in case that is somehow possible.
Specifying a `figure.alt` text looks like it would rather _decrease_ accessibility for this use case here?

Or am I using Glossarium incorrectly somehow, or is there some other way how this could be solved?
Note that I am not blocked by this currently; I was just curious which errors Typst would report when I used `--pdf-standard` for my document.

**Comment by quachpas** [comment]
I would consider this a bug. Probably the implementation should switch to using metadata instead of figures. The only benefit is that figures come with easy references.
